src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -628,11 +632,10 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.WAVPACK;
 		} else if (value.contains("mlp")) {
 			format = FormatConfiguration.MLP;
+		} else if (value.equals("openmg")) {
+			format = FormatConfiguration.ATRAC;
 		} else if (value.contains("atrac")) {
 			format = FormatConfiguration.ATRAC;
-			if (media.getCodecV() == null && !media.getContainer().equals(FormatConfiguration.RM)) {
-				media.setContainer(FormatConfiguration.ATRAC);
-			}
 		} else if (value.equals("jpeg")) {
```

Comment:
@Sami32 Please think of that I do so many different things that I cannot remember all the details. Or, maybe it's just early Alzheimer's - I don't know, but often when I see a case, it's like I "see it for the first time". I don't remember much from I wrote this PR for example, I've written over 20000 lines of unrelated code in the meanwhile.